examples/tv-app/android/App/platform-app/src/main/java/com/matter/tv/server/service/AppPlatformService.java

```diff
@@ -98,17 +118,39 @@ public void setActivity(Activity activity) {
   }
 
   public void addContentApp(ContentApp app) {
-    app.setEndpointId(
-        mAppPlatform.addContentApp(
-            app.getVendorName(),
-            app.getVendorId(),
-            app.getAppName(),
-            app.getProductId(),
-            "1.0",
-            new ContentAppEndpointManagerImpl(context)));
+    int retEndpointId = -1;
+    int desiredEndpointId = discoveredEndpoints.getInt(app.getAppName(), -1);
+    if (desiredEndpointId > 0) {
+      retEndpointId =
+          mAppPlatform.addContentAppAtEndpoint(
+              app.getVendorName(),
+              app.getVendorId(),
+              app.getAppName(),
+              app.getProductId(),
+              "1.0",
```

Comment:
what is "1.0" ? can we add a named constant?

Reply:
This is app version. Plan is to replace this with actual version information later on. Hence did not create a constant for it.